vhdbuilder/packer/test/run-pretest.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
I don't think this was actually necessary, I just needed to force git to change this file to LF (which was failing CI with CRLF). I should just fix my git config, but usr/bin/env is slightly more unix-y